src/test/ui/error-codes/E0106.stderr

```diff
@@ -2,25 +2,49 @@ error[E0106]: missing lifetime specifier
   --> $DIR/E0106.rs:2:8
    |
 LL |     x: &bool,
-   |        ^ expected lifetime parameter
+   |        ^ expected named lifetime parameter
+   |
+help: consider introducing a named lifetime parameter
+   |
+LL | struct Foo<'lifetime> {
+LL |     x: &'lifetime bool,
```

Comment:
Couldn't this use `'a` instead of `'lifetime`, to be more realistic?

Reply:
It could, but we are also targeting people who might be just picking Rust up with these messages who might not know the meaning of `'a` to begin with.

Reply:
It would be nicer to say that `'a` is a lifetime somewhere in the diagnostic, than risk people copy-pasting the `'lifetime` example literally into their code and getting the wrong impression that it's a convention.

Reply:
https://github.com/rust-lang/rust/pull/68583 tackles this